docs/usage.md

```diff
@@ -144,6 +144,20 @@ Then pass this to `osv-scanner` with this:
 osv-scanner --lockfile osv-scanner:/path/to/osv-scanner.json
 ```
 
+## Specify Package(s) to Exclude from OSV Query
+
+If you want to exclude a package or a range of packages from the query to the
+OSV database, you can pass a list of regex patterns. The `osv-scanner` will
+check for packages matching the specified regex patterns and exclude these
+packages from being checked for known vulnerabilities. The exclusions apply to
+both lockfile specified packages and SBOM specified packages.
+
+The exclusion regex patterns can be applied with the following flag:
+
+```bash
+osv-scanner --exclusions="^example.com/.*,^react$" <rest-of-cli-interaction>
```

Comment:
nit: I think it would be better to use single quotes as (at least in Bash) they don't do interpolation and the nature of regex syntax makes me paranoid of that happening by mistake 😅 

```suggestion
osv-scanner --exclusions='^example.com/.*,^react$' <rest-of-cli-interaction>
```

(it might even be worth adding a warning about that?)

Reply:
Bash in general makes me paranoid, so I get it... 🤣

Reply:
Updated.